xml/snmp_sysdescr.xml

```diff
@@ -1039,7 +1051,7 @@
     <param pos="0" name="os.vendor" value="Brocade"/>
     <param pos="0" name="os.product" value="Fabric OS"/>
     <param pos="0" name="os.device" value="Switch"/>
-    <param pos="0" name="os.cpe23" value="cpe:/o:broadcom:fabric_operating_system:-"/>
+    <param pos="0" name="os.cpe23" value="cpe:/o:brocade:fabric_operating_system:-"/>
```

Comment:
Just a note. This change is interesting though correct.   Broadcom purchased Brocade in 2016.   We have a CPE remap because the CPEs were under Broadcom and not Brocade.  It appears that this changed in Sept 2021.

https://nvd.nist.gov/products/cpe/detail/1041182?namingFormat=2.3&orderBy=CPEURI&keyword=Brocade&status=FINAL